### PR TITLE
Newsletters: add 317 (2024-08-23)

### PR DESCRIPTION
**Pull request by harding**
- [ ] Lede, releases/RCs, topic entries @harding 
- [x] Client/Services section @bitschmidty 
- [x] Notable merges @Gustavojfe

### CODE REVIEW

**Comment by bitschmidty** [comment]
Smaller Client/Services section this month, suggestions welcome

**Comment by harding** [comment]
Added a news item since we didn't have any.  It's for a discussion we normally would've covered next week (it started after this PR was initially opened).

**Review by bitschmidty — COMMENTED** [review]
News item looks good to me

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  who use an interactive wireless communication protocol like Bluetooth,
+  the number of round trips doesn't matter.  But, for users who prefer
+  to keep their devices airgapped, each roundtrip requires two manual
+  interventions---which can quickly add up to an annoying amount of work
+  when signing frequently or using multiple devices for scripted
+  multisignatures.
+
+  The disadvantage of this protocol was mentioned by Maxwell in his
+  original description, it "leaves open a [side-channel][topic side
+  channels] that has exponential cost per additional bit, via grinding
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:53]:
```suggestion
  channels] that has an exponential cost per additional bit, via grinding
```

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  [...] but it eliminates the obvious and very powerful attacks where
+  everything is leaked in a single signature.  This is clearly less
+  good, but it's only a two-move protocol, so many places which wouldn't
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:56]:
```suggestion
  good, but it's only a two-move protocol, so many places that wouldn't
```

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+  consider using a countermeasure could pick this up for free just as an
+  element of a protocol spec."
+
+  This protocol is a clear upgrade over not using anti-exfiltration at
+  all and Pieter Wuille [notes][wuille exfil2] that it is probably the
+  best possible anti-exfiltration possble with single-round signing.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:62]:
```suggestion
  best possible anti-exfiltration with single-round signing.
```

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+  identities.
+
+- **Taproot Assets v0.4.0-alpha released:**
+  The [v0.4.0-alpha][taproot assets v0.4.0] release supports the Taproot Assets
+  protocol on mainnet for onchain asset issuance and atomic swaps using
+  [PSBTs][topic psbt] and routing assets through the Lightning Network.
+
+- **Stratum v2 benchmarking tool released:**
+  The initial [0.1.0 release][sbm 0.1.0] supports testing, reporting, and comparing
+  performance of Stratum v1 and Stratum v2 protocols in different mining scenarios.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:98]:
```suggestion
  the performance of Stratum v1 and Stratum v2 protocols in different mining scenarios.
```

**Comment by Gustavojfe** [comment]
just pushed all merge summaries for this week

File: _posts/en/newsletters/2024-08-23-newsletter.md (hunk 1 above)
Comment (harding) [reply _posts/en/newsletters/2024-08-23-newsletter.md:53]: Leaving as-is.  I think this is slightly more invasive a change, for no significant improvement in comprehensibility, than I'd like to make to a quote.

**Comment by harding** [comment]
Made all edits (or left a comment), thanks @bitschmidty and @LarryRuane!  Reviewed @bitschmidty and @Gustavojfe sections, great stuff!  Added lede, checked for updated releases/RCs (none), and added topic entries.  Thanks everyone!

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+  consider using a countermeasure could pick this up for free just as an
+  element of a protocol spec."
+
+  This protocol is a clear upgrade over not using anti-exfiltration at
+  all and Pieter Wuille [notes][wuille exfil2] that it is probably the
+  best possible anti-exfiltration with single-round signing.
+  However, Wuille advocates for the deployed two-round anti-exfiltration
+  protocol to prevent even grinding-based exfiltration.
+
+  Discussion was ongoing at the time of writing.
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:71]:
```suggestion
  Discussion is ongoing at the time of writing.
```

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+  possible future where legacy wallets may no longer be loadable in Bitcoin
+  Core, with [descriptor][topic descriptors] wallets becoming the default. When
+  selecting a wallet to migrate, the GUI will prompt the user to enter the
+  wallet’s passphrase, if it has one.
+
+- [Core Lightning #7540][] improves the formula that calculates the probability
+  of a successful routing through a channel in the `renepay` plugin (see
+  Newsletter [#263][news263 renepay]) by adding a constant multiplier that
+  represents the probability that a randomly chosen channel in the network is
+  able to forward at least 1 msat. The default value is set to 0.98, but this
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:172]:
```suggestion
  represents the probability that a randomly chosen channel in the network
  can forward at least 1 msat. The default value is set to 0.98, but this
```

File: _posts/en/newsletters/2024-08-23-newsletter.md [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+  may be changed in the future after further testing.
+
+- [Core Lightning #7403][] adds a channel filtering payment modifier to the
+  `renepay` plugin which disables channels with a very low `max_htlc`. This can
+  be extended in the future to filter out channels that are undesirable for
+  other reasons: high base fee, low capacity, high latency. In addition, a new
```
Comment (LarryRuane) [inline _posts/en/newsletters/2024-08-23-newsletter.md:178]:
```suggestion
  other reasons: high base fee, low capacity, and high latency. In addition, a new
```

File: _posts/en/newsletters/2024-08-23-newsletter.md (hunk 1 above)
Comment (LarryRuane) [reply _posts/en/newsletters/2024-08-23-newsletter.md:58]: Agreed, sorry, I once again failed to notice that this is inside a quote.

**Review by LarryRuane — COMMENTED** [review]
A few small items... otherwise LGTM

**Comment by bitschmidty** [comment]
Took a suggestion from @LarryRuane, squashed, and merged, thank you all! 🚀

**Pull request merged**